dev/io.openliberty.thirdparty.faces-3.0/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Fragment-Host: io.openliberty.org.apache.myfaces.3.0;bundle-version=1.0
+
+Bundle-Name: WebSphere Jakarta MyFaces Third Party API
```

Comment:
Looks like we're missing the Bundle-Description also we should discuss removing "WebSphere" in 2.3 we have the following "WebSphere JSF MyFaces Third Party API". Discuss what we should use other than WebSphere? Is OpenLiberty better? Maybe "Open Liberty Jakarta Server Faces MyFaces Third Party API"

Reply:
Changed to Open Liberty